### DISCUSSION TITLE
Obtener localidad desde csf.retrieve

### DISCUSSION THREAD

**Discussion by MarmolejoBlad** [SAT-CFDI/python-satcfdi#23] [General]
¿Es posible obtener el dado de Localidad al utilizar la consultaa csf.retrieve(rfc=rfc, ic_cif=if_cif)?
Tanto para persona física como persona moral se obtienen la mayoría de los datos, excepto el dato de localidad (ver imagen de referencia de una persona moral):
![image](https://github.com/user-attachments/assets/f8472cc3-e173-41ea-9a9e-453f52ab395d)

Acá la forma en la que obtengo los datos:
![image](https://github.com/user-attachments/assets/222133cc-dffc-4e34-9e19-5eaaad6e6006)

**Comment by Atjoms** [comment]
No, baiscamente es por que usa una funcion para la validacion de la csf con el QR y extrae los datos desde esa pagina web la url es def  "https://siat.sat.gob.mx/app/qr/faces/pages/mobile/validadorqr.jsf?D1=10&D2=1&D3={id_cif}_{rfc}" lo que hace es entrar a esa eb y extrae los valores que estan presentes, debido a que la web no muestra la localidad ni tampoco las obligaciones fiscales es imposible extraerlas usando esa funcion
